### ISSUE TITLE
Bug: opening links in PDF with brave set to the default browser produces an error.

### ISSUE THREAD

**Issue by jamesray1** [brave/browser-laptop#12715]
### Description
https://askubuntu.com/questions/997511/how-to-open-a-link-in-a-pdf-with-the-error-unable-to-open-external-link-faile

tl;dr: I tried lots of things, but links in PDFs only open when Firefox or Chrome are set to the default browser, but not when Brave is set to the default browswer.

### Steps to Reproduce
Set Brave to the default browser (if it's not already).

Download this PDF [here](http://www.pubcom.com/508%20Links%20Test_2.pdf) and open it in Ubuntu 17.10 (use a VM or install it as a dual boot option), then click a link in the Evince document viewer (the default PDF app).

**Actual result:**
"Unable to open external link. Failed to execute child process “sh” (Permission denied)".

![screenshot from 2018-01-19 11-58-22](https://user-images.githubusercontent.com/16969914/35134587-498d3514-fd2b-11e7-9079-c51de6ead646.png)

**Expected result:**
Should open the link in Brave.

**Reproduces how often:**
100% (on Ubuntu, it works on Linux).

### Brave Version

**about:brave info:**
 
Brave: 0.19.134 
rev: 316cfa522ffe6a558fe9f2f3f299c0c602dfb691 
Muon: 4.5.38 
libchromiumcontent: 63.0.3239.132 
V8: 6.3.292.49 
Node.js: 7.9.0 
Update Channel: Release 
OS Platform: Linux 
OS Release: 4.13.0-30-generic 
OS Architecture: x64

**Reproducible on current live release:**
Yes

### Additional Information
See the Ask Ubuntu page if you like. Here's the link again. https://askubuntu.com/questions/997511/how-to-open-a-link-in-a-pdf-with-the-error-unable-to-open-external-link-faile

**Comment by DrissiReda** [comment]
Not sure if this is a brave issue or not, I've been recently having the same problem with the new firefox quantum, I've tried reproducing the same case you, but with firefox quantum instead of brave. Same result.

**Comment by bsclifton** [comment]
For folks experiencing this, can you please try grabbing the new version of Brave?
https://brave.com/download

If you run still run into problems, let's create an issue in the new repository:
https://github.com/brave/brave-browser

**Comment by jamesray1** [comment]
Looking forward to seeing instructions for installation on Manjaro (Arch) Linux!
